src/Api/SecretsManager/Controllers/ServiceAccountsController.cs

```diff
@@ -67,6 +67,41 @@ public class ServiceAccountsController : Controller
         return new ListResponseModel<ServiceAccountResponseModel>(responses);
     }
 
+    [HttpGet("/service-account/{serviceAccountId}/service-accounts")]
+    public async Task<ServiceAccountResponseModel> GetByServiceAccountIdAsync(
+     [FromRoute] Guid serviceAccountId)
+    {
```

Comment:
The route seems a bit odd. We should be able to just use `{id}` which resolves to `/service-accounts/{id}`. Can we also move this below `CreateAsync`? which keeps the `organizations` routes together.

```suggestion
    [HttpGet("{id}")]
    public async Task<ServiceAccountResponseModel> GetByServiceAccountIdAsync(
     [FromRoute] Guid id)
    {
```